src/scripts/generate-unary-ops.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+import sys
+import os
+import argparse
+import settings
+import collections
+import random
+import subprocess
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate
+
+GENERATED_FILENAME = 'unary-ops-'
+UNARY_OPERATORS = ["+", "-", "++", "--", "!", "~", "typeof", "delete", "void"]
+DEFAULT_TEST_COUNT = 100
+OPERAND_COUNT_MIN = 2
+OPERAND_COUNT_MAX = 11
+DEFAULT_SEED = 10000
+MAX = (2**53) - 1
+MIN = -MAX
+FALSE_VALUES_COUNT = 5
+BITWISE_NOT_MAX = (2**31) - 1
+BITWISE_NOT_MIN = -BITWISE_NOT_MAX
+TEST_CASE_IN_A_FILE = 1000
```

Comment:
Where does this number come from? In #2 this number seems to me `800`. It would be great to unify it.